src/components/tx/GasParams/index.tsx

```diff
@@ -26,6 +26,7 @@ type GasParamsProps = {
   isEIP1559: boolean
   onEdit: () => void
   gasLimitError?: Error
+  willExecute?: boolean
```

Comment:
```suggestion
  isExecution?: boolean
```

Reply:
I think it is a better naming + coherent with `<AdvancedParamsForm />`

Reply:
I just realised that it can be totally removed as we have `isExecution` here as well.